server/src/core/common/pg.py

```diff
@@ -1,35 +0,0 @@
-from typing import List, Dict
-
-import psycopg2
-
-
-class DBConnection:
```

Comment:
Alchemy can not appear anywhere else apart from this file. The whole applicaiton should not care about what DB client we are using. 

I don't like using Alchemy in the applicatoin. It should be used only for DB migration. 

I HATE ORM in general. It is a absolutly unnecessary complexity for any application. 

I suggest you completly remove Alchemy. You only added unnecessary complexity to the project without any benefit